src/Bundler.js

```diff
@@ -208,7 +214,11 @@ class Bundler extends EventEmitter {
     // Emit an HMR update for any new assets (that don't have a parent bundle yet)
     // plus the asset that actually changed.
     if (this.hmr && !isInitialBundle) {
-      this.hmr.emitUpdate([...this.findOrphanAssets(), ...loadedAssets]);
+      if (didProcessHtmlAsset) {
```

Comment:
Can we do the detection of HTML assets in the `HMRServer` instead of here? All of the updated assets are already passed to `emitUpdate` so it should be easy to do the detection in there.

Reply:
I'll do that. thanks